bundles/org.openhab.binding.doorbird/src/main/java/org/openhab/binding/doorbird/internal/api/DoorbirdInfo.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.doorbird.internal.api;
+
+import java.util.ArrayList;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.doorbird.internal.model.DoorbirdInfoDTO;
+import org.openhab.binding.doorbird.internal.model.DoorbirdInfoDTO.DoorbirdInfoBha;
+import org.openhab.binding.doorbird.internal.model.DoorbirdInfoDTO.DoorbirdInfoBha.DoorbirdInfoArray;
+
+import com.google.gson.JsonSyntaxException;
+
+/**
+ * The {@link DoorbirdInfo} holds information about the Doorbird.
+ *
+ * @author Mark Hilbush - Initial contribution
+ */
+@NonNullByDefault
+public class DoorbirdInfo {
+    private @Nullable String returnCode;
+    private @Nullable String firmwareVersion;
+    private @Nullable String buildNumber;
+    private @Nullable String primaryMacAddress;
+    private @Nullable String wifiMacAddress;
+    private @Nullable String deviceType;
+    private @Nullable String controllerId;
+    private ArrayList<String> relays = new ArrayList<>();
```

Comment:
I would make these final if possible

Reply:
I didn't see an easy way to do this without a fair bit of rework. Would prefer to leave as is.